nemo/lightning/megatron_parallel.py

```diff
@@ -1742,7 +1742,7 @@ def __init__(self, validation_step: bool = False, val_drop_last: bool = True) ->
 
     def forward(
         self, batch: Dict[str, torch.Tensor], forward_out: torch.Tensor
-    ) -> Tuple[torch.Tensor, Dict[str, torch.Tensor]]:
+    ) -> Tuple[torch.Tensor, torch.Tensor, Dict[str, torch.Tensor]]:
```

Comment:
not needed?

Reply:
I updated the return type to reflect that we now return three values